definitions/npm/ansi-regex_v4.x.x/flow_v0.25.x-/ansi-regex_v4.x.x.js

```diff
@@ -0,0 +1,10 @@
+declare module 'ansi-regex' {
+  declare export type Options = {
+    /**
+     * Match only the first ANSI escape.
+     */
+    onlyFirst?: boolean,
+  };
```

Comment:
This might be better to make exact and immutable:

```
{|
    /**
     * Match only the first ANSI escape.
     */
    +onlyFirst?: boolean,
  |}
```